### PR TITLE
Add zenodo badge and citation file

### PR DESCRIPTION
**Pull request by leifdenby**
I've set up the zenodo integration for our github organisation which ensures the future releases on github area automatically uploaded to zenodo and given a DOI. I've also made sure that our existing releases on github have been pushed to zenodo (using https://github.com/imcf/zenodo-doi-for-existing-release). I've added a zenodo doi badge to the README and created a  `CITATION.cff` file as zenodo recommend you do (https://citation-file-format.github.io/).

Could you have a look @martinjanssens and let me know if this looks ok to you?

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+authors:
+  - given-names: Leif
+    family-names: Denby
+    email: leif@denby.eu
+    affiliation: University of Leeds
+    orcid: 'https://orcid.org/0000-0002-7611-9222'
+  - given-names: Martin
+    family-names: Janssens
+    email: martin.janssens@wur.nl
+    affiliation: University of Wageningen
```
Comment (martinjanssens) [inline CITATION.cff:19]: Could we change my affiliation to 'Wageningen University & Research'? The uni is being picky about us using its proper name, I'm sorry...

Comment (leifdenby) [reply CITATION.cff]: No problem! :)

Comment (leifdenby) [reply CITATION.cff]: That should be all done now @martinjanssens

**Review by martinjanssens — COMMENTED** [review]
Looks wonderful!

**Review by martinjanssens — APPROVED** [review]

**Pull request merged**